pkg/kubelet/cm/types.go

```diff
@@ -92,6 +93,14 @@ type CgroupManager interface {
 	ReduceCPULimits(cgroupName CgroupName) error
 	// GetResourceStats returns statistics of the specified cgroup as read from the cgroup fs.
 	GetResourceStats(name CgroupName) (*ResourceStats, error)
+	// GetCgroupMemoryLimit returns the memory limit of the specified cgroup as read from cgroup fs.
+	GetCgroupMemoryLimit(name CgroupName) (uint64, error)
+	// GetCgroupCpuLimit returns the cpu quota, cpu period, and cpu shares of the specified cgroup as read from cgroup fs.
```

Comment:
it will be more extendable to return a pre-defined CPU struct instead of individual values of the CPU.

Reply:
i updated the naming of this to reflect that it is getting and setting cgroup config based on upstream review. struct would be overkill.